meiz/index.html

```diff
@@ -3,11 +3,11 @@
 title: Meiz
 ---
 
-{% for file in site.static_files %}
-  {% assign pageurl = page.url | replace: 'index.html', '' %}
-  {% if file.path contains pageurl %}
-    {% if file.extname == '.jpg' %}
-	<img src="{{ site.baseurl }}/{{ file.path }}" style="max-width:30%" />
-    {% endif %}
-  {% endif %}
-{% endfor %}
+{% for file in site.static_files -%}
+  {%- assign pageurl = page.url | replace: 'index.html', '' -%}
+  {%- if file.path contains pageurl -%}
+    {%- if file.extname == '.jpg' %}
+      <img src="{{ site.baseurl }}{{ file.path }}" alt="{{ file.basename }}" style="max-width:30%">
```

Comment:
@WebFreak001
Note how there’s no whitespace control marker on the right-hand side of the `if file.extname…` expression. This preserves the line-break + indentation before the `<img>` tag.